lib/crystalball/logger.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'logger'
+
+module Crystalball
+  class << self
+    # This logs information to the standard output based on the configured log level,
+    # and also logs unfiltered information to the configured log file.
+    def log(severity_sym, *args, &block)
```

Comment:
where are tests for this method?

Reply:
https://github.com/toptal/crystalball/blob/d95d623c62af37a0d26c994e04c91ca881de263d/spec/logger_spec.rb